pcvalidate panics if the file argument doesn't exist

```console
❯ ./pcvalidate/pcvalidate foo
open /home/fabio/dev/publiccode-parser-go/foo: no such file or directory
panic: interface conversion: error is *fs.PathError, not publiccode.ValidationResults

goroutine 1 [running]:
main.hasValidationErrors(...)
	/home/fabio/dev/publiccode-parser-go/pcvalidate/pcvalidate.go:109
main.main()
	/home/fabio/dev/publiccode-parser-go/pcvalidate/pcvalidate.go:96 +0x568
```

--- a/pcvalidate/pcvalidate.go
+++ b/pcvalidate/pcvalidate.go
@@ -112,11 +112,17 @@ func hasValidationErrors(results error) bool {
 		return false
 	}
 
-	for _, res := range results.(publiccode.ValidationResults) {
-		switch res.(type) {
-		case publiccode.ValidationError:
-			return true
+
+	switch e := results.(type) {
+	case publiccode.ValidationResults:
+		for _, res := range e {
+			switch res.(type) {
+			case publiccode.ValidationError:
+				return true
+			}
 		}
+	case error:
+		return true
 	}
 
 	return false

